### PR TITLE
Fix input css as per design

### LINKED ISSUE #841: Fix Add and Remove Vega Token Buttons
When user inputs the number of Vega Tokens to be added or removed from a validator then the CTA button turns White which clashes with white text box for number of Vega Tokens.

Screenshot 2021-11-16 at 13 34 15

Screenshot 2021-11-16 at 13 32 09

### LINKED ISSUE #860: Review button styles on withdraw forms
- copy: remove complete (except where a withdraw is complete and the button is not a button.
- armed state appear disabled

### LINKED ISSUE #864: Wallet sidebar: sticky + scrollable
Problem: mobile
- Long sidebar will be cut off on mobile

screenshot_2021-11-17_at_10 08 26

imperfection: desktop
- if the user has scrolled down the page to interact with the form, they may have also scrolled so that the side bar may not visible

Proposed solution.
The side bar should be held in a container that is fixed to the height of the viewport. (on mobile it is viewport - nav links). Meaning scrolling down the content area keeps the sidebar at top of viewport.
However the side bar content may be longer than the container so it also needs to be scrollble

### PR DESCRIPTION
**Pull request by dexturr**
Closes #860
Closes #841
Closes #864

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -31,6 +34,7 @@ body,
   }
 }
 
+// TODO replace with blueprint themeing
```
Comment (pgriscti) [inline src/index.scss:37]: Is there a follow up ticket for this?

File: src/index.scss [hunk 2]
```diff
@@ -82,8 +87,15 @@ button {
 
   &:disabled,
   &.disabled {
-    opacity: 0.6;
-    cursor: not-allowed;
+    opacity: 1;
+    background: $bg-color-default;
+    color: $color-default;
+    border: 1px solid #696969;
```
Comment (pgriscti) [inline src/index.scss:93]: can we pout the colour in a variable?

File: src/index.scss [hunk 3]
```diff
@@ -159,21 +171,30 @@ button {
...
 input[type="password"] {
   width: 100%;
   padding: 12px;
-  border-radius: 4px;
-  background-color: #3f3f3f;
-  color: white;
-  border: 0;
+  border-radius: 0px;
+  border: 1px solid $white;
+  // background-color: $dark-gray4;
```
Comment (pgriscti) [inline src/index.scss:183]: remove comment

**Review by pgriscti — APPROVED** [review]
few minor followups

File: src/index.scss (hunk 3 above)
Comment (dexturr) [reply src/index.scss:183]:
```suggestion
```

File: src/index.scss (hunk 2 above)
Comment (dexturr) [reply src/index.scss]: good shout

**Pull request merged**